project/AkkaBuild.scala

```diff
@@ -1382,16 +1385,43 @@ object AkkaBuild extends Build {
 
     val cluster = exports(Seq("akka.cluster.*"), imports = Seq(protobufImport()))
 
-    val parsing = exports(Seq("akka.parboiled2.*", "akka.shapeless.*"))
+    val parsing = exports(Seq("akka.parboiled2.*", "akka.shapeless.*"),
+        imports = Seq(optionalResolution("scala.quasiquotes")))
+
+    val httpCore = exports(Seq("akka.http.*"),
+        imports = Seq(streamAndHttpImport("akka.stream.*"),
+            streamAndHttpImport("akka.parboiled2.*"),
+            streamAndHttpImport("akka.shapeless.*")))
+
+    val http = exports(Seq("akka.http.impl.server",
+        "akka.http.scaladsl.server.*", "akka.http.javadsl.server.*",
+        "akka.http.scaladsl.client", "akka.http.scaladsl.coding", "akka.http.scaladsl.common",
+        "akka.http.scaladsl.marshalling", "akka.http.scaladsl.unmarshalling"),
+        imports = Seq(streamAndHttpImport("akka.stream.*"),
+            streamAndHttpImport("akka.http.*"),
+            streamAndHttpImport("akka.parboiled2.*")))
 
-    val httpCore = Nil // FIXME #15689
+    val httpTestKit = exports(Seq("akka.http.scaladsl.testkit.*", "akka.http.javaadsl.testkit.*"),
```

Comment:
No worries about that, this is so easy to get wrong. The problem is that there are no safeguards that would catch a problem like this.